CityworksConnection/Connect2Cityworks.pyt

```diff
@@ -351,6 +351,7 @@ class Tool(object):
                 webmap = WebMap(mapitem)
 
                 for layer in webmap['operationalLayers']:
+                for layer in webmap.layers:
```

Comment:
either line 353 or line 354 must be deleted, or else the indentations need to be fixed.

Reply:
@mottesen-cityworks

Reply:
@mottesen-cityworks Could you review/make this change so I can merge?

Reply:
This has already been fixed in https://github.com/Esri/crowdsource-reporter-scripts/pull/57
You can close https://github.com/Esri/crowdsource-reporter-scripts/pull/53 it's not needed anymore.